server/athenian/api/openapi/openapi.yaml

```diff
@@ -1435,3 +1513,327 @@ components:
         merger:
           $ref: '#/components/schemas/DeveloperSet'
       type: object
+    FilterCommitsRequest:
+      description: Filter for listing commits.
+      example:
+        account: 1
+        date_from: 2000-01-23
+        date_to: 2020-05-23
+        in:
+        - '{1}'
+      allOf:
+      - $ref: '#/components/schemas/CommitFilter'
+      - properties:
+          property:
+            description: Main trait of the commits - the core of the filter.
+            enum:
+            - bypassing_prs
+            type: string
+        required:
+        - property
+        type: object
+    CommitsGroupedByTime:
+      description: Lists of commits for each time interval.
+      example:
+        include:
+          users:
+            github.com/gkwillie:
+              avatar: https://avatars0.githubusercontent.com/u/60340680?v=4
+        data:
+        - date: 2000-01-23
+          commits:
+          - repository: github.com/athenianco/athenian-api
+            hash: 79f0bccb856b35ef08ec93da2ff5585da4d2e50e
+            author:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+            committer:
+              login: github.com/gkwillie
+              name: Groundskeeper Willie
+              email: bot@athenian.co
+              timestamp: 2020-02-25T14:27:21Z
+              message: 'Bump version: 0.0.42 → 0.0.43'
+              size_added: 1
+              size_removed: 1
+              files_changed: 1
```

Comment:
Why are these fields under `committer` instead of being direct children of the `commit`.
imo it should be one indentation less

Reply:
F*cking copy-paste in PyCharm